lib/tapioca/config_builder.rb

```diff
@@ -66,7 +60,7 @@ def merge_options(*options)
     DEFAULT_OPTIONS = T.let({
       "postrequire" => Config::DEFAULT_POSTREQUIRE,
       "outdir" => nil,
-      "generate_command" => default_command,
+      "generate_command" => Config::DEFAULT_COMMAND,
```

Comment:
And, based on the message above, we wouldn't want to set any defaults for this command either.